### ISSUE TITLE
repo.spring.io is disallowing release downloads, breaking build (darcula dependency)

### ISSUE THREAD

**Issue by charles-dyfis-net** [freeplane/freeplane#183]
As per the announcement at https://spring.io/blog/2020/10/29/notice-of-permissions-changes-to-repo-spring-io-fall-and-winter-2020, `repo.spring.io` is only to be used for downloading snapshots and not released software; all released software is to be downloaded from Maven Central.

Right now, this means that attempting to build freeplane with a completely clean cache can fail due to a HTTP 401:

```none
1: Task failed with an exception.
-----------
* Where:
Build file '/build/source/freeplane/build.gradle' line: 61

* What went wrong:
A problem occurred evaluating project ':freeplane'.
> Could not resolve all files for configuration ':freeplane:lib'.
   > Could not resolve com.bulenkov:darcula:2018.2.
     Required by:
         project :freeplane
      > Could not resolve com.bulenkov:darcula:2018.2.
         > Could not get resource 'https://repo.spring.io/plugins-release/com/bulenkov/darcula/2018.2/darcula-2018.2.pom'.
            > Could not GET 'https://repo.spring.io/plugins-release/com/bulenkov/darcula/2018.2/darcula-2018.2.pom'. Received status code 401 from server: Unauthorized
```

...and indeed, as of today, if one visits https://repo.spring.io/plugins-release/com/bulenkov/darcula/2018.2/darcula-2018.2.pom in a web browser, one receives a HTTP 401.

The `build.gradle` should be updated to retrieve this content from Maven Central.

**Comment by charles-dyfis-net** [comment]
Hmm. Not that easy, perhaps:

```
Resource missing. [HTTP HEAD: https://dl.bintray.com/freeplane/freeplane/com/bulenkov/darcula/2018.2/darcula-2018.2.jar]
Failed to get resource: GET. [HTTP HTTP/1.1 401 Unauthorized: https://repo.spring.io/plugins-release/com/bulenkov/darcula/2018.2/darcula-2018.2.pom)]
Resource missing. [HTTP GET: https://repo1.maven.org/maven2/com/bulenkov/darcula/2018.2/darcula-2018.2.pom]
Resource missing. [HTTP HEAD: https://repo1.maven.org/maven2/com/bulenkov/darcula/2018.2/darcula-2018.2.jar]
```

...so we're not on Maven Central, _and_ we're no longer available from spring.io

**Comment by charles-dyfis-net** [comment]
...perhaps mirroring the binary to the `dl.bintray.com` directory is the right immediate choice, then?

**Comment by dpolivaev** [comment]
I think it's already fixed on 1.8.x branch. Which version do you build?

**Comment by charles-dyfis-net** [comment]
I ran into this trying to build the 1.8.10 tag (building a package for NixOS, so I want to stick with released versions + specific backported patches to the extent possible).

**Comment by charles-dyfis-net** [comment]
This does indeed to longer happen on the tip of the 1.8.x branch. Apologies for the noise, I'll bisect back to the commit with the fix and pull it into the package.

**Comment by charles-dyfis-net** [comment]
(For the record, should anyone else run into this: The immediate fix is 36ee57e87f3e2df63fb0d97e7d278aa71e047d32; 8cce354fcc10c1bbbdcf63a30a983cc633ed05de is also relevant)
